lib/atomic/misc.less.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, it, expect } from "vitest";
+import { renderLess } from "../test/less-test-utils";
+
+describe("atomic: misc", () => {
+    it("should output all atomic css classes", async () => {
+        const css = await renderLess(`
+            @import (reference) "./lib/base/internal.less";
```

Comment:
@dancormier I think that those `@import (reference)` if they are necessary to compile the file correctly they should be added to the source code and not to the test file (I am suggesting to do what we have done for colors [here](https://github.com/StackExchange/Stacks/blob/develop/lib/exports/color-mixins.less#L1-L2)). This gives helpful information to the reader of the source code highlighting that the file have dependencies on other less files and cannot be used/compiled in isolation. So essentially I would prefer if possible to follow the same pattern ES Modules are using to import other modules. Let me know if that makes sense to you.

Reply:
Yep, good call. Thanks for the suggestion